lib/Common/Memory/LargeHeapBlock.h

```diff
@@ -29,11 +29,11 @@ struct LargeObjectHeader
 #endif
 
 public:
-    bool markOnOOMRescan;
+    bool markOnOOMRescan:1;
+    bool isExplicitFreed:1;
+    bool isPageHeapAlloc:1;
```

Comment:
removing, introduced in initial design, not referencing this field in many places now

---
In reply to: [60995282](https://github.com/Microsoft/ChakraCore/pull/711#discussion_r60995282) [](ancestors = 60995282)